train_vnext/training.py

```diff
@@ -158,17 +142,85 @@ def process_post_training_csv(csv_path, training_id, classification_name_to_clas
                 prediction_label = PredictionLabel(training_id, 
                                     int(row[0].split('.')[0]), 
                                     classification_name_to_class_id[class_name], 
-                                    float(row[2]), 
-                                    float(row[3]), 
-                                    float(row[4]), 
-                                    float(row[5]), 
+                                    float(row[2])*float(row[6]), 
+                                    float(row[3])*float(row[6]), 
+                                    float(row[4])*float(row[7]), 
+                                    float(row[5])*float(row[7]), 
                                     int(row[6]), 
                                     int(row[7]), 
                                     float(row[8]), 
                                     float(row[9]))
                 payload_json.append(prediction_label)
     return jsonpickle.encode(payload_json, unpicklable=False)
 
+def save_training_session(config, model_location, perf_location, prediction_labels_location):
+    # First, upload the model to blob storage
+    cur_date_time = "{date:%Y_%m_%d_%H_%M_%S}".format(date=datetime.datetime.now())
+    file_name = "frozen_inference_graph_" + cur_date_time + ".pb"
+    model_location = upload_model_to_blob_storage(config, model_location, file_name)
+
+    # Get the mapping for class name to class id
+    overall_average, classification_name_to_class_id = get_average_and_classification_map(perf_location, config.get("tagging_user"), config.get("url"))
+
+    # Create a new training session in db and get its id
+    training_id = construct_new_training_session(perf_location, classification_name_to_class_id, overall_average, cur_date_time, model_location, config.get("tagging_user"), config.get("url"))
+
+    # Upload prediction labels to the db
+    upload_data_post_training(prediction_labels_location, classification_name_to_class_id, training_id, config.get("tagging_user"), config.get("url"))
+
+def upload_model_to_blob_storage(config, model_location, file_name):
+    blob_storage = BlobStorage.get_azure_storage_client(config)
+    uri = 'https://' + config.get("storage_account") + '.blob.core.windows.net/' + config.get("storage_container") + '/' + file_name
+    blob_storage.create_blob_from_path(
```

Comment:
@samiyaakhtar consider adding metadata to the blob like we do [here](https://github.com/CatalystCode/active-learning-detect/blob/a941ea65745320a73887f9fa6faf9769d363f541/functions/pipeline/onboardqueueproccessor/__init__.py#L77). I would include at least  training id and  user_name

Reply:
So training id we get back after we have already uploaded the blob to the storage. I can add the original file path and username maybe>?